assets/launcher/lang/sklmessages_en_US.properties

```diff
@@ -84,6 +84,9 @@ editor.game.link.compat=Read more
 editor.game.modal.label.title=What is Compatibility Mode?
 editor.game.modal.label.body=This option may be required for some specific modpacks, but most of the time it\'s not. Enable this only when you know what you\'re doing!
 
+editor.alert.title=Delete profile
+editor.alert.text=Are you sure want to delete this profile? This action cannot be undone.
```

Comment:
What do you think about:
* "Profile Deletion" instead of "Delete profile"
* "Are you sure **you** want to delete this profile?"?

Reply:
Delete confirmation